.github/workflows/hygiene-check.yml

```
@@ -0,0 +1,173 @@
+          # Extract score
+          HYGIENE_SCORE=$(python -c "
+          import json
+          try:
+              with open('hygiene_report.json', 'r') as f:
+                  data = json.load(f)
+              score = data.get('metrics', {}).get('hygiene_score', 75)
+              print(int(score))
+          except:
+              print(75)
+          ")
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)